### PR TITLE
fix: upgraded versions.tf to include minor bumps from tpg v5

### PR DESCRIPTION
**Pull request by g-awmalik**

### CODE REVIEW

**Comment by g-awmalik** [comment]
@apeabody @bharathkkb - PTAL

File: versions.tf [hunk 1]
```diff
@@ -20,7 +20,17 @@ terraform {
 
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 5.0"
+      version = ">= 3.53, < 6"
+    }
+
+    local = {
+      source  = "hashicorp/local"
+      version = "~> 2.4"
```
Comment (apeabody) [inline versions.tf:28]: Could be considered a breaking change? Came out after the most recent release of this module.

File: versions.tf [hunk 2]
```diff
@@ -20,7 +20,17 @@ terraform {
...
+    }
+
+    null = {
+      source  = "hashicorp/null"
+      version = "~> 3.2"
```
Comment (apeabody) [inline versions.tf:33]: Could be considered a breaking change? Came out after the most recent release of this module.

**Review by apeabody — APPROVED** [review]
Aligned the cloudbuild tests and bumped `terraform-google-modules/project-factory/google` so the tests init can run on TPG v5.

**Pull request merged**